webview/platforms/android/__init__.py

```diff
@@ -218,9 +218,11 @@ def _back_pressed(self, v, key_code, event):
             self.pywebview_window.closed.set()
         return True
 
+    @run_on_ui_thread
     def get_size(self):
         return self.webview.getWidth(), self.webview.getHeight()
 
+    @run_on_ui_thread
```

Comment:
@r0x0r This will return None, `get_url` is not dependent on the ui thread. If you insist on @run_on_ui_thread, use Semaphore to lock it.